nucypher/characters/lawful.py

```diff
@@ -1064,6 +1071,12 @@ def __init__(self,
             message = "Initialized Stranger {} | {}".format(self.__class__.__name__, self)
             self.log.debug(message)
 
+    def __prune_arrangements(self) -> None:
+        result = self.datastore.del_expired_policy_arrangements(
+            now=datetime.fromtimestamp(self._arrangement_pruning_task.clock.seconds()))
+        self.log.debug(f"Pruned {result} policy arrangements.")
```

Comment:
What do you think about only printing this when `result > 0` - no use flooding the logs (even in debug) with useless entries.

Reply:
I pointed out below that the session could rollback and not prune any arrangements but this would still be written to the log.

Reply:
:heavy_check_mark: